links.html

```diff
@@ -22,20 +22,13 @@
 
 
 <body>
-<div class="Width-wrap">
-<Width-wrap>
+<div class="width-wrap">
 <header>
     <nav>
         <ul>
             <li> <a href="index.html">Home</a>  </li>
             <li> <a href="links.html">Links </a>  </li>
             <li> <a href="glossary.html">Glossary </a>  </li>
-            <li> <a href="https://jigsaw.w3.org/css-validator/" target = "blank">CSS Validator</a>  </li>
-            <li> <a href="https://developer.mozilla.org/en-US/" target = "blank">Devloper Tools</a>  </li>
-            <li> <a href="https://caniuse.com/" target = "blank">CanIuse</a>  </li>
-            <li> <a href="https://necolas.github.io/normalize.css/8.0.1/normalize.css" target = "blank">Normalized CSS</a>  </li>
-            <li> <a href="https://fontpair.co/" target = "blank">Font Pair</a>  </li>
-            <li> <a href="https://fonts.google.com/" target = "blank">Google Fonts</a>  </li>
         </ul>
```

Comment:
These links should be in the 'useful links' list below, not in the navigation header